mvpiframework/src/main/java/com/coreyhorn/mvpiframework/MVPISettings.kt

```diff
@@ -0,0 +1,5 @@
+package com.coreyhorn.mvpiframework
+
+object MVPISettings {
```

Comment:
should this sit in app? Is it accessible from app to set it?

Reply:
I believe it should be accessible from the app.